dnf/util.py

```
@@ -60,7 +60,11 @@ def ensure_dir(dname):
         if not os.path.isdir(dname):
             raise IOError("%s is not a directory" % dname)
     else:
-        os.makedirs(dname, mode=0o755)
+        try:
+            os.makedirs(dname, mode=0o755)
+        except OSError as e:
+            if e.errno != os.errno.EEXIST:
```

please add also the ```os.path.isdir``` test because this error is being ignored now